### DISCUSSION TITLE
Why My GitHub Action Was Disabled ?

### DISCUSSION THREAD

**Discussion by TheHritu** [community/community#60000] [Actions]
### Select Topic Area

Question

### Body

Its Showing "GitHub Actions is currently disabled for your account. Please reach out to GitHub Support for assistance." 
Tell me it's reason & solution also.

**Comment by KaramiMostafa** [comment]
I'm sorry to hear that :(
There can be several reasons for that, but most probably, it's due to security issues. This could be related to suspicious activity on your account. It's better to contact [GitHub Support](https://support.github.com/request) directly.
